docs/rest.md

```diff
@@ -0,0 +1,622 @@
+# Snappy Ubuntu Core REST API
+
+Version: 1.0.1 DRAFT (look for "not implemented" to find bits still in flux or
+not implemented)
+
+## Versioning
+
+As the API evolves, some changes are deemed backwards-compatible (such
+as adding methods or verbs, or adding members to the returned JSON
+objects) and don’t warrant an endpoint change; some changes won’t be,
```

Comment:
I think it would be clearer like: some changes won't be backwards-compatible, ...